crates/rpc/src/v04/method/estimate_message_fee.rs

```diff
@@ -0,0 +1,42 @@
+use pathfinder_common::{BlockId, CallParam, ContractAddress, EntryPoint, EthereumAddress};
+use stark_hash::Felt;
+
+use crate::context::RpcContext;
+use crate::v02::method::call::FunctionCall;
+use crate::v02::types::reply::FeeEstimate;
+use crate::v03::method::estimate_message_fee::EstimateMessageFeeError;
+
+#[derive(serde::Deserialize, Debug, PartialEq, Eq)]
+pub struct EstimateMessageFeeInput {
+    message: MsgFromL1,
+    sender_address: EthereumAddress,
```

Comment:
This is redundant (`sender_address` is now in `MsgFromL1` as `from_address`).

Reply:
Fixed